README.md

```diff
@@ -1 +1,166 @@
-# My-Portfolio
\ No newline at end of file
+<a name="readme-top"></a>
+
+
+
+<div align="center">
+  
+  <br/>
+
+  <h3><b>My Portfolio README</b></h3>
+
+</div>
+
+# 📗 Table of Contents
+
+- [📖 About the Project](#about-project)
+  - [🛠 Built With](#built-with)
+    - [Tech Stack](#tech-stack)
+    - [Key Features](#key-features)
+- [💻 Getting Started](#getting-started)
+  - [Setup](#setup)
+  - [Prerequisites](#prerequisites)
+  - [Install](#install)
+  - [Usage](#usage)
+  - [Run tests](#run-tests)
+- [👥 Authors](#authors)
+- [🤝 Contributing](#contributing)
+- [⭐️ Show your support](#support)
+- [🙏 Acknowledgements](#acknowledgements)
+- [📝 License](#license)
+
+
+# 📖 [My Portfolio] <a name="about-project"></a>
+
+
+
+**This is a project in my first milestone for the process of creating a portfolio website, I have set up a new repository and prepared it for development using best practices (as linters). I have build the first two sections that is the header and the headline section**
+
+## 🛠 Built With <a name="built-with"></a>
+ 
+ ### Tech Stack <a name="tech-stack"></a>
+- **HTML5**
+- **CSS3**
+### Key Features <a name="key-features"></a>
+
+
+- **This decribes who i am to my employer**
+
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+
+## 💻 Getting Started <a name="getting-started"></a>
+
+
+To get a local copy up and running, follow these steps.
+
+### Prerequisites
+
+In order to run this project you need:
+
+- HTML, CSS, and Javascript know-how.
+- Git installation
+- A text editor
+- A terminal
+- A web browser to view output e.g Google Chrome
+- An IDE e.g Visual studio code 
+
+### Setup
+
+Clone this repository to your desired folder:
+
+Commands:
+
+```sh
+  cd hello microverse
+  git clone git@github.com:Eccb7/My-Portfolio.git
+```
+
+### Install
+
+Install this project with:
+
+Command:
+
+```sh
+  cd Hello
+  gem install
+```
```

Comment:
- [ ] Kindly customise this section correctly as well to make your project look more professional. this is an [example template](https://github.com/rychrr/portofolio_page/blob/main/README.md) you can use to set up this section :+1: 
[OPTIONAL]
- [ ] you also missed some sections and its descriptions which the initial reviewers should have flagged, check this  [example template](https://github.com/rychrr/portofolio_page/blob/main/README.md) and fix them accordingly :+1: